wurst/file/ByteBuffer.wurst

```diff
@@ -0,0 +1,145 @@
+package ByteBuffer
+
+import Table
+import Bitwise
+
+/*	ByteBuffer is a utility class that allows you to write/read bytes, shorts and integer to/from it.
+	It may be useful for compacting bytes into integers or decomposing integers into bytes.
```

Comment:
Is there a use case for writing bytes and reading bytes, or is this a type conversion utility?

Reply:
There is. Base64Encoder/Decoder use it to write bytes there and read bytes from there to give these bytes to the user then.